Inputs/GraphQL/gqlQueries.robot

```diff
@@ -67,3 +67,18 @@ getAHUsPropertiesOfSpecificGroup
     [Arguments]    ${group_name}
     ${query}=  set variable  query getGroupsAHUProperties{site{groups: children(selector: {type: Group, name: "General-test"}){name ahus:children(selector: {type: AHU}){ name CoolSource: propString(name: "CoolSource") DesignCapacity: propFloat(name: "DesignCapacity") DesignCop: propFloat(name: "DesignCop")}}}}
     return from keyword    ${query}
+
+#========Queries related to Alarm testcases
+
+getAlarmMsgAHUFailToTurOff
```

Comment:
This can be replaced with getAlarmStatusQuery [line no:30]. Only need to add 'message' to the existing query.
query alarmStatus{ alarms(selector:{subjectName: "General-test", type : AhuFailToTurnOff}) { type severity status message }}
Thus the query will be generic and can use for other Alarms as well.

Reply:
Done